python_ta/cfg/graph.py

```diff
@@ -95,13 +104,16 @@ class CFGBlock:
     predecessors: List[CFGEdge]
     # This block's out-edges (to blocks that can execute immediately after this one).
     successors: List[CFGEdge]
+    # Whether there exists a path from the start block to this block.
+    is_unreachable: bool
```

Comment:
Generally prefer positive instead of negative versions of attributes. So here, use "reachable" instead of "unreachable" as the attribute name (this is also consistent with the description written above). Also you can drop the "is" prefix, and just use "reachable".